README.md

```diff
@@ -1,20 +1,22 @@
-# Fleek Non-Fungible Apps
 
-**The repository for Fleek Non-Fungible Apps project**
+# Non-Fungible Apps - by Fleek
 
 > 🚧 IMPORTANT 🚧 - This initiative is under development, so this repo should be treated as a WIP. The goals and the roadmap might change as the project is shaped.
 
 ## ⚡ Overview
 
-This is the landing point for Fleek's initiative to implement infrastructure as Solidity contracts.
+This is the landing point for our initiative to decentralize web3 app frontend infrastructure, their distribution, and access, through Non-fungible Applications, an NFT-based implementation.

```

Comment:
The current implementation supports frontends, but the long-term goal is to add support for everything an app has (as much as we can at least, like multi-sigs and such)
Should that be included here?

Reply:
the word infrastructure here I think covers it, but in the one below I will do edit